lib/args.js

```diff
@@ -0,0 +1,65 @@
+"use strict";
+
+/**
+ * Argv command flags.
+ */
+var path = require("path");
+var _ = require("lodash");
+var nopt = require("nopt");
+var chalk = require("chalk");
+
+// Option flags.
+var FLAGS = {
+  // Global: Should apply across any action.
+  general: {
+    builderrc: {
+      desc: "Path to builder config file (default: `.builderrc`)",
+      types: [path]
+    }
+  },
+
+  run: {
+    retries: {
+      desc: "Number of times to re-run task on non-zero exit (default: `1`)",
+      types: [Number]
+    }
```

Comment:
Slightly confusing: the wording makes me think the default is to try twice (1 initial attempt, 1 retry). But looking at the retry code I think the default is 1 attempt and no retries. IMO either this should be named `--attempts` with a default of 1, or have `--retries` default to 0 (add 1 to whatever number is given, for the initial attempt).